api/translator/src/main/java/com/project/translator/adapter/in/web/MessageController.java

```diff
@@ -35,6 +38,12 @@ MessageDomain getById(@PathVariable("id") Long id) {
         return messageUseCase.getMessageById(id);
     }
 
+
+    @GetMapping(value = "/search/language")
```

Comment:
Może lepiej usunąć ten człon search z path'a? Do tej pory mamy wszystkie metody dostępne z path'em message/{path_param}. To dodanie dwóch członów wydaje mi się nieintuicyjne - zmieniłbym na messages/language?value={request_param}

Reply:
ok, "search" usuniete